### PR TITLE
ci: preview from fork

### PR DESCRIPTION
**Pull request by NoamGaash**
# Description
<Please describe the suggested changes>

## screenshots
<paste here>

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          name: dist
+          path: dist
+  
+      - name: Save PR number
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          mkdir -p ./pr
+          echo $PR_NUMBER > ./pr/pr_number
+      - uses: actions/upload-artifact@v4
```
Comment (zushar) [inline .github/workflows/build.yml:46]: Is there a reason you used a different version than the rest?
i think v3 is the latest stable version

**Review by zushar — APPROVED** [review]
Thank you so much for asking me to help, I really appreciate it. And even more appreciates your work for the community

File: .github/workflows/build.yml (hunk 1 above)
Comment (NoamGaash) [reply .github/workflows/build.yml:46]: Hmm, I didn't notice the version - I just copy & pasted it from somewhere online. Thanks for noticing that.
Actually, seems like v3 is about to be deprecated
https://github.com/actions/upload-artifact?tab=readme-ov-file#actionsupload-artifact
image

I guess we could make an issue to upgrade the upload thing everywhere


Thanks for reviewing this!

Comment (NoamGaash) [reply .github/workflows/build.yml:46]: I'll merge this PR, let's open an issue if any further improvements required

**Pull request merged**